### PR TITLE
Cockpit as RPM and not container anymore in Dolomite

### PR DESCRIPTION
**Pull request by tbazant**
### Description

Cockpit is distributed as an RPM package in Dolomite and not as podman container image anymore

### Are there any relevant issues/feature requests?

* bsc#1217518

### CODE REVIEW

File: tasks/cockpit-installation.xml [hunk 1]
```diff
@@ -20,38 +22,39 @@
     <abstract>
       <para>
         Even though in most cases &cockpit; is present on &productnameshort; by
-        default, but there may be some cases when &cockpit; is missing. This
-        topic provides instructions how to install it.
+        default, there may be certain cases when &cockpit; is missing. This
+        topic provides instructions how to install it and enable its service.
```
Comment (dariavladykina) [inline tasks/cockpit-installation.xml:26]:
```suggestion
        topic provides instructions on how to install it and enable its service.
```

**Review by dariavladykina — COMMENTED** [review]
Just a minor nit from me, thanks!

File: tasks/cockpit-installation.xml [hunk 2]
```diff
@@ -20,38 +22,39 @@
...
     </para>
+<screen os="alp-dolomite">&prompt.sudo;<command>transactional-update pkg install cockpit-ws</command></screen>
   </section>
-  <section xml:id="task-example-executing">
-    <title>Installing &cockpit;</title>
+  <section xml:id="cockpit-installing-additional-plugins">
+    <title>Installing additional &cockpit; plugins</title>
     <para>
-      Though &cockpit; is present in the pre-built images by default, the
-      plugin for administration of virtual machines needs to be installed
```
Comment (lvicoun) [inline tasks/cockpit-installation.xml:42]: Please keep this for SLE Micro as it was as it was a specific demand to mention this particular plugin.

File: tasks/cockpit-installation.xml [hunk 3]
```diff
@@ -20,38 +22,39 @@
...
     </abstract>
   </info>
-  <section xml:id="task-example-introduction">
+  <section xml:id="cockpit-installation-introduction">
     <title>Introduction</title>
     <para>
-      &cockpit; is included in the delivered pre-built images by default, or
-      can be installed if you are installing your own instances manually. For
-      details regarding the manual installation, refer to the
+      &cockpit; is included in the delivered pre-built images by default. If your scenario requires
```
Comment (lvicoun) [inline tasks/cockpit-installation.xml:33]: This does not apply to older Micro releases, the previous version of the text was more pecise.

**Review by lvicoun — CHANGES_REQUESTED** [review]

**Comment by fsundermeyer** [comment]
Replaced by #286

**Pull request closed without merging**